README.md

```diff
@@ -29,7 +31,18 @@ Instantiate an instructor named 'Robin' and call its `givesHomework` method with
 Your Answer:
 
 ```js
-// your code here
+
+var Instructor = function(name, assignment) {
+  this.name;
+  this.assignment;
```

Comment:
Not Quite, we need to assign a value to each of these properties on the constructor function` Instructor`.  For example,
```

function Instructor(name){
  this.name = name;
  this.givesHomework = function(assignment){
    console.log(this.name + " gives the students " + assignment + " for Friday's homework.");
  }
}
```